src/components/PageList.vue

```diff
@@ -14,7 +14,7 @@
 				trailing-button-icon="close"
 				:show-trailing-button="isFilteredView"
 				@trailing-button-click="clearFilterString" />
-			<NcActions class="toggle toggle-push-to-right">
+			<NcActions v-if="!isPublic" class="toggle toggle-push-to-right">
```

Comment:
```suggestion
			<NcActions v-if="currentCollectiveCanEdit" class="toggle toggle-push-to-right">
```

Public collectives shares can be read-write, so there's no reason not to edit templates in public shares. But I agree that we should only show the button if the collective is editable.